ci/rspec-audit/migratedb

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
```

Comment:
Do we still need this file? isn't it a double migration now?

Reply:
Not since this script file is running only on audit engine tests.
The audit engine tests not running the DB migration files because Conjur container is not require for the engine tests.